tasks/main.yml

```diff
@@ -35,6 +35,23 @@
     name: dell-system-update
     state: present
 
+- name: create /usr/libexec/dell_dup
+  file:
+    path: /usr/libexec/dell_dup
+    state: directory
+    mode: '0755'
+  when: dell_dsu_rollout_gpg_keys
+
+- name: copy gpg keys
+  copy:
+    dest: /usr/libexec/dell_dup/{{ item }}
+    src: "{{ item }}"
+  loop:
+    - 0x756ba70b1019ced6.asc
```

Comment:
Some of these keys are in small letters and some in capitals.
I see that that's how they are published on https://linux.dell.com/files/pgp_pubkeys/ == not an issue :)

First I thought it would be nice to not have write the shortish fingerprint of the GPG key in the task file and just get em all from a subdirectory. But now I think it's nice to have the short name written down somewhere in here and not just a blob of GPG key.